src/components/story/story.service.ts

```diff
@@ -312,19 +350,24 @@ export class StoryService {
     if (filter.name) {
       query.call(filterByString, label, 'name', filter.name);
     }
-    const listResult: {
-      items: Array<{
-        identity: string;
-        labels: string[];
-        properties: Story;
-      }>;
-      hasMore: boolean;
-      total: number;
-    } = await runListQuery(query, input, secureProps.includes(input.sort));
+    query.call(addAllSecureProperties, ...secureProps).with(
+      `
+          {
+            ${addBaseNodeMetaPropsWithClause(baseNodeMetaProps)},
+            ${listWithSecureObject(secureProps)}
+          } as node
+        `
```

Comment:
Just an FYI we have a [`mapping`](https://github.com/SeedCompany/cord-api-v3/blob/master/src/core/database/mapping.helper.ts) function that could make this easier
```suggestion
      mapping('node', {
        ...addBaseNodeMetaPropsWithClause(baseNodeMetaProps),
        ...listWithSecureObject(secureProps),
      })
```
Those two other functions would need to be replaced with ones that return objects instead of strings.

Reply:
Got it.